adminSiteServer/apiRoutes/mdims.ts

```diff
@@ -22,9 +67,10 @@ export async function handleMultiDimDataPageRequest(
     }
     const rawConfig = req.body as MultiDimDataPageConfigRaw
     const id = await createMultiDimConfig(trx, slug, rawConfig)
+
     if (
         FEATURE_FLAGS.has(FeatureFlagFeature.MultiDimDataPage) &&
-        (await isMultiDimDataPagePublished(trx, slug))
+        (await multiDimDataPageExists(trx, { slug, published: true }))
     ) {
```

Comment:
This code path here should also call `validateGrapherSlug`

Reply:
Hmm, true. I didn't realize we are checking only for published graphers in the validation. That's even more complex.